trueclass.rb

```diff
@@ -0,0 +1,65 @@
+class TrueClass
+  #  Document-class: TrueClass
+  #
+  #  The global value <code>true</code> is the only instance of class
+  #  TrueClass and represents a logically true value in
+  #  boolean expressions. The class provides operators allowing
+  #  <code>true</code> to be used in logical expressions.
+  # 
+  #
+  # call-seq:
+  #   true.to_s   ->  "true"
+  #
+  # The string representation of <code>true</code> is "true".
+  #
+  def to_s
+    Primitive.attr! 'inline'
+    Primitive.cexpr! 'rb_cTrueClass_to_s'
+  end
+
+  alias_method :inspect, :to_s
+
+  #
+  #  call-seq:
+  #     true & obj    -> true or false
+  #
+  #  And---Returns <code>false</code> if <i>obj</i> is
+  #  <code>nil</code> or <code>false</code>, <code>true</code> otherwise.
+  #
+  def &(obj)
+    Primitive.attr! 'inline'
+    Primitive.cexpr! 'RTEST(obj)?Qtrue:Qfalse;'
```

Comment:
If the `cexpr!` even necessary here? Did you bench it against pure ruby?, e.g. `obj ? true : false`.

Reply:
No, I'll try it.

Reply:
pure ruby impl case as fast as using `cexpr!`
So, replace it in ruby code.